docs/source/users/getting_started.rst

```diff
@@ -1,5 +1,193 @@
+.. _getting-started:
+
+###############
 Getting Started
-===============
+###############
+
+|Python 2.7| will be needed for running/installing this. Instructions
+for setting up python are available
+`here <https://github.com/mantidproject/fitbenchmarking/wiki/Setting-up-Python>`__.
+
+1. Download this repository or clone it using
+   `git <https://git-scm.com/>`__:
+   ``git clone https://github.com/mantidproject/fitbenchmarking.git``
+2. Open up a terminal (command prompt) and go into the
+   ``fitbenchmarking`` directory.
+3. Once you are in the right directory, type
+   ``python setup.py install``.
+4. Install mantid, platform specific instructions
+   `here <https://github.com/mantidproject/fitbenchmarking/wiki/Installing-Mantid>`__.
```

Comment:
Same as above. Linked page is now in this file.

Reply:
Also fixed